docs/ref/templates/builtins.txt

```diff
@@ -720,6 +720,10 @@ included template and returns an empty string.
     been evaluated and rendered* - not blocks that can be overridden by, for
     example, an extending template.
 
+.. versionchanged:: 1.9
+
+    Parsed template objects are cached during rendering.
```

Comment:
I am not sure this annotation adds much value as the change doesn't really affect users of the tag, does it?